doc/api/pypeit.rst

```diff
@@ -49,6 +49,7 @@ Submodules
    pypeit.telescopes
    pypeit.tracepca
    pypeit.utils
+   pypeit.version
```

Comment:
This probably should be removed.  I think you can have sphinx ignore the file by editing line 59 of `doc/Makefile` to:

```
	SPHINX_APIDOC_OPTIONS=$(SPHINXAPIOPT) $(SPHINXAPI) --separate -o ./api ../pypeit ../pypeit/tests/* ../pypeit/scripts/* ../pypeit/deprecated/* ../pypeit/version.py
```

Reply:
good catch. made this change.